### DISCUSSION TITLE
GMP collector  unable to connect to POD running with ASM istio sidecar

### DISCUSSION THREAD

**Discussion by sreejesh-radhakrishnan-db** [GoogleCloudPlatform/prometheus-engine#967] [Q&A]
similar to https://github.com/GoogleCloudPlatform/prometheus-engine/discussions/340 but no solution there

We are having a similar issue where ISTIO is running on 15090 and scrape away all the metric. but my app exposing on 8084 is not getting through to GPM.
I followed your step above and I could see this error :

Get "http://240.13.206.101:8084/actuator/prometheus": read tcp xxx.xxx.xxx.xxx:37516->xxx.xxx.xx.xxx:8084: read: connection reset by peer

basically collector not able to connect to the Application POD.

any help please?

also observed this error on pod -istio sidecar container

"textPayload": "[2024-05-10T15:48:49.862Z] "- - -" 0 NR filter_chain_not_found - "-" 0 0 0 - "-" "-" "-" "-" "-" - - 240.13.xxx.xxx:8084 240.13.xxx.xxx:51396 - -",

**Comment by pintohutch** [comment]
@sreejesh-radhakrishnan-db this looks like a duplicate of #966. I'm going to close this to focus on answering there.
